<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DesuraTools</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="accountNameContainer">
      <item>
       <widget class="QLabel" name="desuraAccountName_label">
        <property name="text">
         <string>Desura Account Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="desuraAccountName_input">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="placeholderText">
         <string>Your Desura Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="desuraAccountName_verify">
        <property name="text">
         <string>Verify Account</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="steamID_label">
        <property name="text">
         <string>Steam ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="steamID_input">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="installGamesTab">
         <attribute name="title">
          <string>Owned Desura Games</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QListWidget" name="installGames_list"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="addToSteam_tab">
         <attribute name="title">
          <string>Installed Desura Games</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QListWidget" name="addToSteam_list"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="gameInfoContainer">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>GameIcon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>GameName</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="desuraButton">
        <property name="text">
         <string>Desura Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="installButton">
        <property name="text">
         <string>Install</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGeneratre_Desura_Report"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Selected_to_Steam">
   <property name="text">
    <string>Install Selected</string>
   </property>
  </action>
  <action name="actionGeneratre_Desura_Report">
   <property name="text">
    <string>Generate Desura Report</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
